contracts/utils/structs/EnumerableMap.sol

```diff
@@ -31,6 +31,11 @@ import "./EnumerableSet.sol";
  * - `uint256 -> address` (`UintToAddressMap`) since v3.0.0
  * - `address -> uint256` (`AddressToUintMap`) since v4.6.0
  * - `bytes32 -> bytes32` (`Bytes32ToBytes32`) since v4.6.0
+ *
+ *    // WARNING!
+ *       // delete function should not be used. It will corrupt the data structure,
+ *       // clear function should be used instead.
+ *       // ref https://github.com/ethereum/solidity/pull/11843
```

Comment:
Format is not correct.

See `utils/Address.sol` for how to properly format a warning.

Reply:
Also, delete in not a "function", and there is no `clear` function in this structure.

I would suggest something like:

> Trying to delete such a structure from storage will likely result in data corruption, rendering the structure unusable.
> See https://github.com/ethereum/solidity/pull/11843 for more info
>
> In order to clean an EnumerableMap, you can either remove all elements one by one or create a fresh instance using an array of EnumerableMap.

Reply:
Changed wording and format.
maybe add an issue at contracts/utils/cryptography/MerkleProof.sol to make Warning's Format consistent ?